examples/hydra/README.md

```diff
@@ -0,0 +1,8 @@
+Setup a CI on an Openstack cloud for a Nix repository in one command: `terraform apply`
```

Comment:
Are the no other dependencies for the system to run?

Do I need to configure my openstack credentials? do I need to install some software or setup my environment in any other way?